docs/quickstart.md

```diff
@@ -97,9 +97,9 @@ Update `index.ts` to use `figlet` in the `fetch` handler.
 
   const server = Bun.serve({
     fetch() {
-+     const body = figlet.textSync('Bun!');
++     const body = figlet.textSync("Bun!");
```

Comment:
change here to double to make it consistent with other strings

Reply:
This PR is legendary. Very good work.